tachyon/zk/r1cs/constraint_system/quadratic_arithmetic_program.h

```diff
@@ -126,9 +126,11 @@ class QuadraticArithmeticProgram {
 
     CHECK_GE(domain->size(), matrices.num_constraints);
 
-    std::pmr::vector<F> a(domain->size());
-    std::pmr::vector<F> b(domain->size());
-    std::pmr::vector<F> c(domain->size());
+    std::vector<Evals> abc;
+    abc.reserve(3);
```

Comment:
```suggestion
    std::vector<Evals> abc(3);

Reply:
This will construct evals as if I did `abc.resize(3)`. Behavior is different from `reserve()`.